aw_server/api.py

```diff
@@ -287,46 +264,33 @@ class HeartbeatResource(Resource):
 
     Might be badly suited for:
      - Has activity occurred? (aw-watcher-afk)
+       Edit: Not anymore, aw-watcher-afk has now been rewritten to use heartbeats.
```

Comment:
I'm going to, just wanted to reflect a little more about why I thought it would be badly suited before I did.